packages/eslint-plugin/README.md

```diff
@@ -100,27 +115,28 @@ const reatomFood = (config: {
 }
 ```
 
-If there is an identifier `name` defined in the function scope, unit names must use it as namespace. Otherwise, namespace must be equal to the name of the factory function.
+Object and domain fragments in names may be used together:
+
+```ts
+atom(0, `${name}.atomsRec.field`)
+atom(0, 'Some.atomsRec.field')
+```
 
-For private atoms, `_` prefix can be used:
+You may prefix unit names with `_` to indicate that they are not exposed from factories that create them (this makes Reatom inspector hide them):
 
 ```ts
 const secretState = atom(0, '_secretState')
 ```
 
-You can also ensure that `atom` names have a prefix or a postfix through the configuration, for example:
+You can also ensure prefixes and suffixes for `atom` names through the configuration:
 
 ```ts
-{
+;({
```

Comment:
```suggestion
({
```
Can we delete this semicolon?

Reply:
prettier adds these if `semi` option is false to prevent code like `foo\n({})` from being parsed as `foo({})`